Class2.cs

```diff
@@ -0,0 +1,51 @@
+using UniversityLibrary;
```

Comment:
```suggestion
using System;
```
Без этого не компилируется.